assets/js/src/form_editor/blocks/convert_alignment_to_margin.ts

```diff
@@ -0,0 +1,12 @@
+
+function convert(alignment: 'left' | 'center' | 'right'): string {
```

Comment:
Just an idea. What about using [typescript string enum](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) for the alignment?